src/Common/OvercommitTracker.cpp

```diff
@@ -8,10 +8,11 @@ using namespace std::chrono_literals;
 
 constexpr std::chrono::microseconds ZERO_MICROSEC = 0us;
 
-OvercommitTracker::OvercommitTracker()
+OvercommitTracker::OvercommitTracker(std::mutex & global_mutex_)
```

Comment:
`global_mutex` is unclear name -- what does it mean? We have a lot of "global" objects.